### ISSUE TITLE
Crash when changing tempo with high quality (v3.1.3)

### ISSUE THREAD

**Issue by MartinHotmann** [audacity/audacity#2597]
**Describe the bug**
When I change the tempo of a (stereo AC3) source in high quality it crashes without any message after about 10min.
I have a highend PC, but since the audio is 90min it takes about 25min to process it, but fails 100% of about 5 trys.

**To Reproduce**
1. open AC3 from `.mka`
2. change tempo (select high quality)
3. let it run for about 10 min
4. See (no) error but force close of the app

**Expected behavior**
No crash, no force close but processing the file

**Additional information (please complete the following information):**
 - OS: Windows 11
 - Version 3.1.3 (Chocolatey install)

**Additional context**
I tested it 5 times, it must be something that is reproducable. If reqired I can provide the `.mka` file, but privately would be recommended.

**Comment by petersampsonaudacity** [comment]
I cannot reproduce this on W10 with 3.1.3

**Comment by LWinterberg** [comment]
I have trouble following step 3 as it finishes processing within 5 minutes for me :) In other words, I can't reproduce it either

Are you noticing anything strange about your computer when doing this? like, is your drive/CPU/RAM hitting its limits or something like that?

**Comment by petersampsonaudacity** [comment]
> I have trouble following step 3 as it finishes processing within 5 minutes for me :) In other words, I can't reproduce it either
> 
> Are you noticing anything strange about your computer when doing this? like, is your drive/CPU/RAM hitting its limits or something like that?

Mine finishes quickly if I don't check "on" the "high quality" option.

When I tested yesterday with a 90 minute stereo track it took 42 minutes to complete on my fairly high-end W10 laptop.

**Comment by LWinterberg** [comment]
right, last time I was trying with a sine, this time I tried with noise and it took significantly longer. 

Regardless - no freeze, crash, error or anything like that for me
